examples/knative/powercli/kn-pcli-nsx-tag-sync/handler.ps1

```diff
@@ -108,7 +108,12 @@ Function Process-Handler {
       $vm = Get-VM -name $vmname | Select-Object Name, PersistentId
    }
    catch {
-      Write-Host "$(Get-Date) - ERROR: unable to retrieve VM object"
+      Write-Host "$(Get-Date) - ERROR: unable to retrieve VM object. Error category: $($error.exception[0].errorcategory)"
```

Comment:
nit: I suggest changing these two `ERROR` to `WARNING`. I did that in my code since it's not necessarily an error and can cause false alarm.

Reply:
fixed